src/features/pages/pages.utils.ts

```
@@ -0,0 +1,39 @@
+    case 'custom':
+      if (!customSort) {
+        throw new Error('Custom sort function is required when sortBy is "custom"')
+      }
+
+      return [...pages].sort(customSort)
+    case 'manual':
+      if (!manualOrder || manualOrder.length === 0) {
+        throw new Error('Manual order array is required when sortBy is "manual"')
+      }
+
+      const orderMap = new Map(manualOrder.map((id, index) => [id, index]))
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)